src/metabase/embedding/models/theme.clj

```diff
@@ -0,0 +1,30 @@
+(ns metabase.embedding.models.theme
+  "Model for embedding themes for use in the embedding theme editor."
+  (:require
+   [metabase.models.interface :as mi]
+   [metabase.models.serialization :as serdes]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(methodical/defmethod t2/table-name :model/EmbeddingTheme [_model] :embedding_theme)
+
+(doto :model/EmbeddingTheme
+  (derive :metabase/model)
+  (derive :hook/entity-id)
```

Comment:
Entity ID is required by the backend test `All exported models should get entity id except those with other unique property (like name)`